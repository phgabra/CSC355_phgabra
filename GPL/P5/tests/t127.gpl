int int_array[3];

int k = 42 + int_array[0];

