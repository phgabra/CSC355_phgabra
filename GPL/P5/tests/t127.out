gpl.cpp::main()
  input file(tests/t127.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int int_array[0] = 0
int int_array[1] = 0
int int_array[2] = 0
int k = 42
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
